jobs/db_store.go

```diff
@@ -351,10 +382,10 @@ func (s *DatabaseStore) UpdateJob(job Job) error {
 	return s.Transaction(func(s *DatabaseStore) error {
 		if res, err := s.conn.Exec(`
 			UPDATE jobs
-				 SET params = $1, log = $2, status = $3
-			 WHERE id = $4
-				 AND instance_id = $5
-		`, string(paramsBytes), string(logBytes), job.Status, string(job.ID), string(job.Instance)); err != nil {
+				 SET params = $1, result = $2, log = $3, status = $4
```

Comment:
oh, the complaint was spaces vs tabs. to be fair, it's probably my vim messing it up.